src/statsig/constants.ts

```diff
@@ -116,4 +116,10 @@ export const DynamicConfigs = {
       maxSlippagePercentage: '0.3',
     },
   },
+  [StatsigDynamicConfigs.CICO_TOKEN_INFO]: {
+    configName: StatsigDynamicConfigs.CICO_TOKEN_INFO,
+    defaultValues: {
+      cicoOrder: {} as { [key: string]: number },
```

Comment:
Wonder if we should do `{ <tokenId>: { <propName>: <propValue> }}` instead? This will make the current dynamic config value more verbose (`cicoOrder` will be repeated on every token), but the benefit is that this will match the format of `/tokensInfo` endpoint and we can just deep merge with it if we want to have more dynamic overrides for any token metadata. We could also make dynamic config name more generic to allow any metadata, but maybe it's best to make them more single purpose (like cico specific stuff in one config) so if at all we have a bunch of overrides, the number of rules doesn't explode. Thoughts? cc @cajubelt @jophish @jeanregisser